src/app/pages/StartPage/StartPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import FlowerIconSmall from '../../components/assets/FlowerIconSmall';
+import LetsGoIcon from '../../components/assets/LetsGo';
+import Typography from '../../components/Typography/Typography';
+import styles from './StartPage.module.css';
+import { Link } from 'react-router-dom';
+
+export default function Start(): JSX.Element {
+  return (
+    <main className={styles.container}>
+      <div className={styles.content}>
+        <FlowerIconSmall />
+        <Typography size="l" className={styles.text}>
+          Nimm Dir Zeit für das, was oft zu kurz kommt:
+        </Typography>
+        <Typography size="l">
+          Zeit für Dich, <br /> Zeit für Gutes
```

Comment:
> It is useful for writing a poem or an address, where the division of lines is significant.
-- [MDN `<br>`: The Line Break element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br)

So, if you consider this a poem go right ahead, otherwise I recommend using two headlines :)